lib/commands/hasFocus.js

```diff
@@ -27,19 +27,21 @@
  */
 
 let hasFocus = function (selector) {
-    let result = this.execute(function (selector) {
-        var focused = document.activeElement
+    return this.elements(selector).then((res) => {
```

Comment:
we looked at `isExisting` and `isSelected` pattern and copied that but yes, if selector arg is passed in there is no need to call this.elements(selector).  -- thanks.